.github/workflows/release_candidate.yml

```
@@ -4,11 +4,23 @@ on:
   push:
     branches:
       - 'changeset-release/main'
+      - 'changeset-release/next-major'
```

```suggestion
      - 'changeset-release/**'
```

Not necessary but these could be `**`

---

I wanted to be explicit here because I'm referencing those branch names directly in the workflow `if` statements